src/AppBundle/EventListener/RedirectToPreferredLocaleListener.php

```diff
@@ -52,8 +52,8 @@ class RedirectToPreferredLocaleListener
     public function __construct(UrlGeneratorInterface $urlGenerator, $locales, $defaultLocale = null)
     {
         $this->urlGenerator = $urlGenerator;
+        $this->locales = array_filter(explode('|', trim($locales)));
```

Comment:
I think you should consult my exchanges with @voronkovich , which explains why a filter has been added.